Assets/dll/libavcodec.so.57

```diff
@@ -0,0 +1 @@
+../../Assets/dll/libavcodec.so
```

Comment:
Why symlink? You control the filename.

Reply:
This is because libavcodec.so is what we want to version control (even though it's binary, at least we'd know consistently when it was changed and by whom), whereas the .57 is the particular version(?) the linker decided it wanted to look for. The symlink is just to redirect it to the actual lib